delete.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# "---------------------------------------------------------"
+# "-                                                       -"
+# "-  Delete uninstalls Cassandra and deletes              -"
+# "-  the GKE cluster                                      -"
+# "-                                                       -"
+# "---------------------------------------------------------"
+
+# Do not set errexit as it makes partial deletes impossible
```

Comment:
Could you elaborate on this comment too?

Reply:
Yes I put my understanding in the comment.